platforms/java/com/area9innovation/flow/HttpServerSupport.java

```diff
@@ -175,10 +175,18 @@ public void handle(HttpExchange exchange) throws IOException
 
 		private static String readInputStream(InputStream stream)
 		{
-			return
-				new BufferedReader(new InputStreamReader(stream))
```

Comment:
Did this implementation also assume that the InputStream was UTF8?

Reply:
removed excplicit encoding to be the same as before - default charset will be used